test/sharedSteps.ts

```diff
@@ -44,16 +43,22 @@ import { getServicesDir, getSpellsDir, getSrcPath } from "../src/lib/paths.js";
 
 import {
   multiaddrs,
+  MY_SERVICE_NAME,
   RUN_DEPLOYED_SERVICES_TIMEOUT,
   WORKER_SPELL,
 } from "./constants.js";
 import { assertHasWorkerAndAnswer, fluence } from "./helpers.js";
 import { validateDeployedServicesAnswerSchema } from "./validators/deployedServicesAnswerValidator.js";
+import { validateSpellLogs } from "./validators/spellLogsValidator.js";
 import {
   validateWorkerServices,
   type WorkerServices,
 } from "./validators/workerServiceValidator.js";
 
+export function getTestAquaDirPath() {
+  return join("test", "_resources", "aqua");
```

Comment:
Should it be kept as a constant in `src/lib/const` ?

Reply:
I moved it in `test/constants.ts`